src/ApiManagement/ApiManagement.ServiceManagement/ApiManagementClient.cs

```diff
@@ -1077,7 +1078,10 @@ byte[] DownloadFileFromLink(string url)
                     headerValue = fromFile ? ContentFormat.SwaggerJson : ContentFormat.SwaggerLinkJson; 
                     break;
                 case PsApiManagementApiFormat.OpenApi:
-                    headerValue = fromFile ? ContentFormat.Openapijson : ContentFormat.OpenapiLink;
+                    headerValue = fromFile ? ContentFormat.Openapi : ContentFormat.OpenapiLink;
```

Comment:
For the case that `specificationApiFormat == PsApiManagementApiFormat.OpenApi` and `fromFile == true`. The return of `GetContentFormatForApiImport`will change from `ContentFormat.Openapijson` to `ContentFormat.Openapi`. Is this a bug fix or a new feature? If it's a new feature, will this affect the exist user script?

Reply:
@solankisamir

Reply:
This is a bug fix. When `OpenApi` was specified, we should only have accept `YAML` files. 
For OpenApi 3.0 document in Json format, we have the format `OpenApiJson`